Makefile.fuzz

```diff
@@ -24,29 +24,34 @@ FUZZ:=$(dir $(wildcard plugin/*/fuzz.go)) # plugin/cache/
 PLUGINS:=$(foreach f,$(FUZZ),$(subst plugin, ,$(f:/=))) # > /cache
 PLUGINS:=$(foreach f,$(PLUGINS),$(subst /, ,$(f))) # > cache
 
+.PHONY: vendor
+vendor:
+	@echo "Vendoring code till go-fuzz supports Go modules!"
+	@go mod vendor
+
 .PHONY: echo
-echo:
+echo: vendor
```

Comment:
this change break the fuzz build

(I also find this confusing, the sooner gofuzz supports, go modules the better). For now it's probably best to not do this

Reply:
sure .. it was sketchy to me too.. but it put it just as an idea